keras_hub/src/models/causal_lm.py

```diff
@@ -339,6 +339,33 @@ def preprocess(x):
         def generate(x):
             return generate_function(x, stop_token_ids=stop_token_ids)
 
+        def strip_prompt_fn(x, prompt):
```

Comment:
Can we change `_fn` to `_function` to make it consistent with other function names in this class (like [this](https://github.com/keras-team/keras-hub/blob/b098a0ba4b25ab479e356ce28a0d6beb6413ba43/keras_hub/src/models/causal_lm.py#L129))?

Reply:
OK